src/commands/projects/add-pull-request.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * This command adds a pull request to a GitHub project column.
+ *
+ * @see: https://octokit.github.io/rest.js/#api-Projects-createCard
+ * const result = await octokit.projects.createCard({column_id, note, content_id, content_type})
+ * /projects/columns/:column_id/cards
+ */
+const flow = require('lodash.flow')
+
+const commonYargs = require('../../../lib/common-yargs')
+const printOutput = require('../../../lib/print-output')
+const authenticatedOctokit = require('../../../lib/octokit')
+
+/**
+ * yargs builder function.
+ *
+ * @param {import('yargs').Yargs} yargs - Instance of yargs
+ */
+const builder = yargs => {
+	const baseOptions = flow([
+		commonYargs.withToken,
+		commonYargs.withJson,
+	])
+	return baseOptions(yargs)
+		.option('column_id', {
+			describe: 'Project column ID',
+			demandOption: true,
+			type: 'number'
+		})
+		.option('pull_request_id', {
```

Comment:
Similar theme here, perhaps pull request URL or GitHub's short syntax would be better to support:

```
--pull-request https://github.com/Financial-Times/next-video-page/pulls/208
--pull-request Financial-Times/next-video-page#208
```